src/ToolBox/SOS/lldbplugin/debugclient.cpp

```diff
@@ -922,6 +921,19 @@ DebugClient::GetThreadContextById(
     dtcontext = (DT_CONTEXT*)context;
     dtcontext->ContextFlags = contextFlags;
 
+    GetContextFromFrame(frame, dtcontext);
+    hr = S_OK;
+
+exit:
+    return hr;
+}
+
+// Internal function
+void
+DebugClient::GetContextFromFrame(
+    lldb::SBFrame& frame,
```

Comment:
Is the byreference passing of SBFrame just a performance optimization, or is the value actually modified anywhere? 

If it is just a performance optimization, it may be nice to mark all the byrefererence arguments with `const` to make the intent clear.